axolotl-web/src/main.js

```diff
@@ -35,8 +36,21 @@ Vue.directive('longclick', longClickInstance)
 Vue.component('font-awesome-icon', FontAwesomeIcon)
 Vue.use(VueChatScroll)
 Vue.use(BootstrapVue)
+Vue.mixin({
+  methods: {
+    linkify: function (content) {
+      return this.$sanitize(LinkifyHtml(content))
+    },
+  },
+})
+var defaultSanitizeOptions = {
+    allowedTags: ['a', 'b'],
```

Comment:
Are the `<b>` tags actually sent by Signal?

Reply:
All tags are sent by signal, though none of them as actual HTML. 

The official signal clients pass through everything that the user sends. From their perspective no sent tags should be interpreted, they should all be plaintext.

Reply:
Yeah nanu answered me that the tags were added by the linkify plugin just above. I was wondering if we could find a solution where we don't need v-html but that seems unlikely if we want to be able to display links inside the message. However @nanu-c I think you can remove every "allowedTags" in the SanitizeOptions if you go it the other way around (`linkify(sanitize(message.Message))` instead of `sanitize(linkify(message.Message))`). Can you please do that?

Reply:
I reversed the order now,  i think it's ready now